src/components/wizard/wizard.component.ts

```diff
@@ -115,8 +115,13 @@ export class WizardComponent implements AfterViewInit, OnDestroy {
 
     @ContentChild('footerTemplate', { static: false }) footerTemplate: TemplateRef<WizardFooterContext>;
 
+    private _subscriptions: Subscription[] = [];
+
     id: string = `ux-wizard-${uniqueId++}`;
     invalidIndicator: boolean = false;
+    currentStep: any;
+    completed: boolean = false;
+    subscription: Subscription;
```

Comment:
Not sure if you need both `subscription` and `subscriptions` here? I think only one can be active at a time because the user can only click next on one step at a time, so probably drop the array.